nautobot_circuit_maintenance/views.py

```diff
@@ -218,51 +229,9 @@ def get_extra_context(self, request, instance):
             "parsednotification": parsednotification,
         }
 
-
-class CircuitMaintenanceEditView(generic.ObjectEditView):
-    """View for editting circuit maintenances."""
-
-    queryset = models.CircuitMaintenance.objects.all()
-    model_form = forms.CircuitMaintenanceForm
-
-
-class CircuitMaintenanceDeleteView(generic.ObjectDeleteView):
-    """View for deleting circuit maintenances."""
-
-    queryset = models.CircuitMaintenance.objects.all()
-
-
-class CircuitMaintenanceBulkImportView(generic.BulkImportView):
-    """View for bulk of circuit maintenances."""
-
-    queryset = models.CircuitMaintenance.objects.all()
-    table = tables.CircuitMaintenanceTable
-
-
-class CircuitMaintenanceBulkEditView(generic.BulkEditView):
-    """View for bulk editing circuitmaintenance features."""
-
-    queryset = models.CircuitMaintenance.objects.all()
-    filterset = filters.CircuitMaintenanceFilterSet
-    table = tables.CircuitMaintenanceTable
-    form = forms.CircuitMaintenanceBulkEditForm
-
-
-class CircuitMaintenanceBulkDeleteView(generic.BulkDeleteView):
-    """View for bulk deleting circuitmaintenance features."""
-
-    queryset = models.CircuitMaintenance.objects.all()
-    filterset = filters.CircuitMaintenanceFilterSet
-    table = tables.CircuitMaintenanceTable
-
-
-class CircuitMaintenanceJobView(generic.ObjectView):
-    """Special View to trigger the Job to look for new Circuit Maintenances."""
-
-    queryset = models.CircuitMaintenance.objects.all()
-
-    def get(self, request, *args, **kwargs):
-        """Custom GET to run a the Job."""
+    @action(detail=False, methods=["get"], url_path="job", url_name="circuitmaintenance_job")
```

Comment:
The `circuitmaintenance_` part of the url gets added automatically so we need to change this to `url_name="job"`

```suggestion
    @action(detail=False, methods=["get"], url_path="job", url_name="job")
```